internal/resolution/variable_sources/crdconstraints/crd_constraints_test.go

```diff
@@ -180,74 +180,74 @@ var _ = Describe("CRDUniquenessConstraintsVariableSource", func() {
 				olmentity.NewBundleEntity(bundleSet["bundle-3"]),
 				[]*olmentity.BundleEntity{},
 			),
-			bundles_and_dependencies.NewBundleVariable(
+			bundlesanddependencies.NewBundleVariable(
 				olmentity.NewBundleEntity(bundleSet["bundle-4"]),
 				[]*olmentity.BundleEntity{},
 			),
-			bundles_and_dependencies.NewBundleVariable(
+			bundlesanddependencies.NewBundleVariable(
 				olmentity.NewBundleEntity(bundleSet["bundle-5"]),
 				[]*olmentity.BundleEntity{},
 			),
-			bundles_and_dependencies.NewBundleVariable(
+			bundlesanddependencies.NewBundleVariable(
 				olmentity.NewBundleEntity(bundleSet["bundle-6"]),
 				[]*olmentity.BundleEntity{},
 			),
-			bundles_and_dependencies.NewBundleVariable(
+			bundlesanddependencies.NewBundleVariable(
 				olmentity.NewBundleEntity(bundleSet["bundle-7"]),
 				[]*olmentity.BundleEntity{
 					olmentity.NewBundleEntity(bundleSet["bundle-8"]),
 					olmentity.NewBundleEntity(bundleSet["bundle-9"]),
 					olmentity.NewBundleEntity(bundleSet["bundle-10"]),
 				},
 			),
-			bundles_and_dependencies.NewBundleVariable(
+			bundlesanddependencies.NewBundleVariable(
 				olmentity.NewBundleEntity(bundleSet["bundle-8"]),
 				[]*olmentity.BundleEntity{},
 			),
-			bundles_and_dependencies.NewBundleVariable(
+			bundlesanddependencies.NewBundleVariable(
 				olmentity.NewBundleEntity(bundleSet["bundle-9"]),
 				[]*olmentity.BundleEntity{},
 			),
-			bundles_and_dependencies.NewBundleVariable(
+			bundlesanddependencies.NewBundleVariable(
 				olmentity.NewBundleEntity(bundleSet["bundle-10"]),
 				[]*olmentity.BundleEntity{},
 			),
-			bundles_and_dependencies.NewBundleVariable(
+			bundlesanddependencies.NewBundleVariable(
 				olmentity.NewBundleEntity(bundleSet["bundle-14"]),
 				[]*olmentity.BundleEntity{},
 			),
-			bundles_and_dependencies.NewBundleVariable(
+			bundlesanddependencies.NewBundleVariable(
 				olmentity.NewBundleEntity(bundleSet["bundle-15"]),
 				[]*olmentity.BundleEntity{},
 			),
-			bundles_and_dependencies.NewBundleVariable(
+			bundlesanddependencies.NewBundleVariable(
 				olmentity.NewBundleEntity(bundleSet["bundle-16"]),
 				[]*olmentity.BundleEntity{},
 			),
 		}
 		variables, err := crdConstraintVariableSource.GetVariables(ctx, entitySource)
 		Expect(err).ToNot(HaveOccurred())
-		Expect(len(variables)).To(Equal(26))
-		var crdConstraintVariables []*crd_constraints.BundleUniquenessVariable
+		Expect(variables).To(HaveLen(26))
+		var crdConstraintVariables []*crdconstraints.BundleUniquenessVariable
 		for _, variable := range variables {
 			switch v := variable.(type) {
-			case *crd_constraints.BundleUniquenessVariable:
+			case *crdconstraints.BundleUniquenessVariable:
 				crdConstraintVariables = append(crdConstraintVariables, v)
 			}
 		}
-		Expect(len(crdConstraintVariables)).To(Equal(11))
+		Expect(crdConstraintVariables).To(HaveLen(11))
 		Expect(crdConstraintVariables).To(WithTransform(CollectGlobalConstraintVariableIDs, ContainElements([]string{
```

Comment:
[ConsistOf ](https://onsi.github.io/gomega/#consistofelement-interface)should help avoid the length check here